WalletWasabi/Blockchain/TransactionBuilding/BnB/ExactMatchStrategy.cs

```diff
@@ -0,0 +1,34 @@
+namespace WalletWasabi.Blockchain.TransactionBuilding.BnB;
+
+public class ExactMatchStrategy : ISearchStrategy
```

Comment:
Do we need this class if we have a working `PruneStrategy`? Or this is here for only comparassion?

Reply:
I think it can be good for perf comparison with the other strategy. But then it can be removed I guess.